CMP1903_A1_2324/Program.cs

```diff
@@ -14,6 +15,28 @@ static void Main(string[] args)
              * Create a Game object and call its methods.
              * Create a Testing object to verify the output and operation of the other classes.
              */
+
+            /// <summary> Main method. </summary>
+            /// <remarks> Creates a Game object and calls its methods. Creates a Testing object to verify the output and operation of the other classes. </remarks>
+            /// <param name="game"> The game object. </param>
+            /// <param name="test"> The testing object. </param>
+            /// <param name="x"> The amount of dice to roll. </param>
+            /// <param name="game"> The game object. </param>
+
+            Game game = new Game();
+            Testing test = new Testing();
+
+            Console.WriteLine("How many dice would you like to roll..?");
+            int x = Convert.ToInt32(Console.ReadLine());
+            
+            Console.WriteLine();
+            
+            test.testDie();
```

Comment:
The test method here rolls the dice 1000 times and test them itself.
The values of the dice have just been rolled by the user were not tested.
(It is because the your Testing class and your Game class create and work on 2 different Die objects).
Maybe your idea is to make testing separate from rolling the dice, but I think the testing function should be used to test every roll the user makes (even you know the use of Random class never generates invalid output, your testing function will verify that (Mark clarified it in his reply to my email)).